src/ranch_acceptors_sup.erl

```diff
@@ -43,7 +44,7 @@ init([Ref, NumAcceptors, Transport]) ->
 	ranch_server:set_addr(Ref, Addr),
 	Procs = [
 		{{acceptor, self(), N}, {ranch_acceptor, start_link, [
-			Ref, N, LSocket, Transport, Logger
+			Ref, (N rem NumConnsSups)+1, LSocket, Transport, Logger
```

Comment:
Don't do this here, you don't know if this'll be correct. Keep `AcceptorId` and pass it to `ranch_server:get_connections_sup/2` like before, and do this calculation in that function instead.